taxonomy.php

```diff
@@ -16,7 +16,7 @@
 $taxonomy = get_queried_object();
```

Comment:
Thinking loud.. have you tried the above code instead?
```suggestion
$taxonomy = get_queried_object()->taxonmy;
```

Reply:
I think I don't understand this suggestion.
If the value of `$taxonomy` is `get_queried_object()->taxonomy` then `$taxonomy` becomes a string and we cannot access `$taxonomy->name` or `$taxonomy->description`

Reply:
You are correct! I saw only this part https://github.com/greenpeace/planet4-master-theme/pull/2425/files#diff-0a1d6d9a542b4540027ce80be62d507972d828c8b530869ac2743b056a2664d4L18